docs/function/functions.md

```diff
@@ -66,6 +66,7 @@ SELECT UNICODE("AmrDeveloper")
 | NOW               |                           | DateTime | Return current date time in `YYYY-MM-DD HH:MM:SS` format.                  |
 | DAYNAME           | Date                      | Text     | Returns the name of the day given a timestamp.                             |
 | MONTHNAME         | Date                      | Text     | Returns the name of the month given a timestamp.                           |
+| HOUR              | DateTime                  | Integer  | Returns the hour given a timestamp.                                        |
```

Comment:
Please use the same description from the website, because its from DateTime type not timestamp

`Return the hour part of a datetime`